python/src/aiconfig/editor/client/src/components/GlobalParametersContainer.tsx

```diff
@@ -0,0 +1,76 @@
+import { createStyles, Container, Accordion, Text } from "@mantine/core";
+import { JSONObject } from "aiconfig";
+import { memo, useCallback, useState } from "react";
+import ParametersRenderer, { ParametersArray } from "./ParametersRenderer";
+
+type Props = {
+  initialValue: JSONObject;
+  onUpdateParameters: (newParameters: any) => void;
+};
+
+const useStyles = createStyles((theme) => ({
+  parametersContainer: {
+    [theme.fn.smallerThan("sm")]: {
+      padding: "0 0 200px 0",
+    },
+    paddingBottom: 50,
+  },
+}));
+
+export default memo(function GlobalParametersContainer({
+  initialValue,
+  onUpdateParameters,
+}: Props) {
+  const [isParametersDrawerOpen, setIsParametersDrawerOpen] = useState(false);
```

Comment:
What's the LOE to make this component UI into the card with bottom show/hide like in the figma -- https://www.figma.com/file/7YVa6KpEDgptj8DiAc5BaU/AiConfig-Editor?type=design&node-id=328-5037&mode=design&t=h3xAGY47py1mg4Fk-0 ?

Reply:
Probably not very high. I can do that as a follow-up next week